### PR TITLE
checkstyle

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

**Comment by sils** [comment]
not fully tested yet

File: bears/general/AnnotationBear.py [hunk 1]
```diff
@@ -17,14 +17,17 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
...
-        :param language:        Language to be whose annotations are to be
-                                searched.
-        :param coalang_dir:     external directory for coalang file.
-        :return:                HiddenResults containing a dictionary with
-                                keys as 'strings' or 'comments' and values as
-                                a tuple of SourceRanges of strings and
-                                a tuple of SourceRanges of comments
-                                respectively.
+        :param language:
+            Language to be whose annotations are to be searched.
```
Comment (nemaniarjun) [inline bears/general/AnnotationBear.py]: Doesn't make complete sense, Incorrect grammer?

File: bears/general/QuotesBear.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+
+class QuotesBear(LocalBear):
+
+    BEAR_DEPS = {AnnotationBear}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
```
Comment (nemaniarjun) [inline bears/general/QuotesBear.py]: ASCIINEMA ?

**Comment by abhsag24** [comment]
ack fac086e hope it still works :P

File: bears/general/QuotesBear.py (hunk 2 above)
Comment (sils) [inline bears/general/QuotesBear.py]: I was too lazy and they make good newcomer issues :P

File: bears/general/AnnotationBear.py (hunk 1 above)
Comment (sils) [inline bears/general/AnnotationBear.py]: filed a newcomer issue, this is just reformatted and not changed. #994

**Comment by sils** [comment]
reack a4dd7a2

**Comment by sils** [comment]
reack https://github.com/coala/coala-bears/pull/992/commits/07c655591b61e865d9192cc52317e3c096084060

**Comment by sils** [comment]
reack https://github.com/coala/coala-bears/pull/992/commits/b42ac399ff0814f7d3d79d6f5ab9031bc5c8fe1d

**Comment by sils** [comment]
@Adrianzatreanu can you debug this on windows?

File: bears/java/CheckstyleBear.py [hunk 3]
```diff
@@ -3,8 +3,8 @@
 
 
 known_checkstyles = {
-    "google": "https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml",
-    "sun": 'https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/sun_checks.xml',
+    "google": "https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-7.2/src/main/resources/google_checks.xml",
+    "sun": 'https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/checkstyle-7.2/resources/sun_checks.xml',
```
Comment (jayvdb) [inline bears/java/CheckstyleBear.py]: `master/src/checkstyle-7.2` ??

-> `checkstyle-7.2/src/main` ?

**Comment by jayvdb** [comment]
unack 3820ffe38

**Comment by sils** [comment]
yeah, not fixing it yet :/

**Comment by sils** [comment]
@Makman2 can you test run this on windows?

**Comment by sils** [comment]
@jayvdb had a way better fix

**Comment by jayvdb** [comment]
Bumping the version to 7.2 would still be useful.

**Pull request closed without merging**